accord-core/src/main/java/accord/local/Command.java

```diff
@@ -730,7 +734,7 @@ public void accept(SafeCommandStore safeStore)
                 if (cur == null)
                 {
                     // need to load; schedule execution for later
-                    safeStore.execute(this, this);
+                    safeStore.execute(this, this).begin(AsyncCallbacks.throwOnFailure());
```

Comment:
Since this is async, what does `throwOnFailure` really mean here?

Reply:
currently it means that calling the callback with a failure throws an exception, presumably to be handled by the threads uncaught exception handler. Having now put that in writing, that doesn't seem like the best approach, since there are other use cases for submitting a failure (ie: in the case of accord Timeouts).

Semantically, it means "start this chain, and I don't care about the result, exception or otherwise", I'll change the name to noop and have it log exceptions

Reply:
This seems quite similar to the pattern of invoking with the `node.agent()`. I'm down with standardising this to some global singleton that handles propagating exceptions, but let's do it across the board. Perhaps we want to call it `logErrors` or something else, that makes it clearer what's happening? Though don't hate `noOp`, and don't mind terribly.